1.4/Source/VanillaPsycastsExpanded/Paths/Archotechist/Ability_AggressiveHeatDump.cs

```diff
@@ -12,16 +12,21 @@
 
     public class Ability_AggressiveHeatDump : Ability
     {
+        public override float GetRadiusForPawn() =>
+            Mathf.Ceil(Mathf.Min(this.pawn.psychicEntropy.EntropyValue / 20f, 9f * base.GetRadiusForPawn()));
+
+        public override float GetPowerForPawn() => DamageDefOf.Flame.defaultDamage *
```

Comment:
Please don’t hardcode references like this. (Yes, I know that’s how it worked before but that does mean I have to like it)